cmake/FindFFTW.cmake

```diff
@@ -0,0 +1,25 @@
+# From: https://github.com/wjakob/layerlab/blob/master/cmake/FindFFTW.cmake
+# FIXME: License concerns?
```

Comment:
The license is `BSD 2-clause "Simplified" License`.  This and the BSD 3-clause are GPL compatible and very common for cmake scripts.

Reply:
I'd be fine with either a BSD or BSD 2-clause license for the CMake files.